containers/kubernetes/how-to/manage-allowed-ips.mdx

```diff
@@ -0,0 +1,50 @@
+---
+meta:
+  title: Managing allowed IP addresses for Kubernetes products
+  description: Learn how to manage allowed IP addresses for your Kubernetes Kapsule and Kosmos clusters. Configure access restrictions with our step-by-step guide.
+content:
+  h1: Managing allowed IP addresses for Kubernetes products
+  paragraph: Learn how to manage allowed IP addresses for your Kubernetes Kapsule and Kosmos clusters. Configure access restrictions with our step-by-step guide.
+tags: kubernetes kapsule kosmos
+dates:
+  validation: 2024-11-05
+  posted: 2024-11-05
+categories:
+  - containers
+---
+
+Restricting IPs on Kubernetes Kapsule or Kosmos clusters enhances security by limiting access to only trusted sources, thereby reducing the risk of unauthorized access and potential attacks.
+This control ensures that only specific IP addresses or networks can interact with your clusters, providing an additional layer of protection.
+The default entry `0.0.0.0/0` enables any host to establish a connection.
+
+<Macro id="requirements" />
+
+- Scaleway account logged into the [Scaleway console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing actions in the intended Organization
+- [Created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule or Kosmos cluster
+
+<Message type="note">
+  Allowed IP configuration is available for public networks only.
+</Message>
+
+## How to add an IP address
+
+1. Click **Kubernetes** in the **Containers** section of the [Scaleway console](https://console.scaleway.com). The **Kubernetes Kapsule dashboard** appears.
```

Comment:
```suggestion
1. Click **Kubernetes** in the **Containers** section of the [Scaleway console](https://console.scaleway.com). The **Kubernetes dashboard** appears.
```

Reply:
We're specifying Kubernetes Kapsule here, but the requirements say that users can do this either on a Kapsule or Kosmos cluster. We must either remove the Kapsule mention here or state that the settings we show on this page are specifically for Kapsule, no?

Reply:
We may leave it to **Kubernetes dashboard**, indeed yes.